### PR TITLE
Update spawnpoints.updated every 6 hours

### PR DESCRIPTION
**Pull request by MonkeyTedBaker**
Only when sightings are saved.

### CODE REVIEW

**Comment by evilmoses** [comment]
"Only when sightings are saved." Do you not mean when sightings are NOT saved?

**Comment by evilmoses** [comment]
Also, am I being blind, or can I not see any reference to the conf setting to indicate when this should be triggered?

Lastly, if this updates all spawnpoints, doesn't that make the clean up utterly useless as it will never delete anything?

**Comment by MonkeyTedBaker** [comment]
I meant only when sightings are saved. When a sighting is saved, corresponding spawnpoint's updated will be updated (every 6 hours).

So basically, during cleanup, we only need to check spawnpoints with updated older than e.g. 24 hr.

File: scan.py [hunk 1]
```diff
@@ -128,7 +128,8 @@ def exception_handler(loop, context):
 
 def cleanup(overseer, manager):
     try:
-        overseer.print_handle.cancel()
+        if hasattr(overseer, 'print_handle'):
+            overseer.print_handle.cancel()
```
Comment (MonkeyTedBaker) [inline scan.py]: This is just a bug (from original Monocle) fix.

**Comment by evilmoses** [comment]
Ah I get you, so it will update the spawnpoint updated, but not insert into sightings?

**Comment by evilmoses** [comment]
I would therefore amend the cleanup sql in this same PR to change the timestamp to 24 hours.

**Comment by MonkeyTedBaker** [comment]
@evilmoses updated the sql. Tested it myself by can you help me with a double check? Thanks.

**Comment by evilmoses** [comment]
I haven't tested it, but see no issue with it. Should work perfectly.

**Pull request merged**